### PR TITLE
Update branding

### PR DESCRIPTION
**Pull request by sbose78**
according to https://docs.google.com/document/d/1P3iAQGDsis6rYPoxBL5GO9UuD1USl2JcXwJf_A1qvvo/edit#

### CODE REVIEW

**Comment by baijum** [comment]
@sbose78 Please fix the lint errors.

File: manifests/devconsole/0.1.0/devconsole-operator.v0.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -26,8 +26,32 @@ spec:
...
+    Kubernetes and OpenShift primitives to allow developers to focus on
+    their application development.
+    
+    Within the Developer perspective there are these main areas:
+
+    * Add - The place to create and build the application using one of this method:
+    
+      - Importing source code from Git
+      - Deploying an existing image
+      - Browse a catalog to deploy or connect application serves
```
Comment (Avni-Sharma) [inline manifests/devconsole/0.1.0/devconsole-operator.v0.1.0.clusterserviceversion.yaml]: application services?

Comment (sbose78) [reply manifests/devconsole/0.1.0/devconsole-operator.v0.1.0.clusterserviceversion.yaml]: I had copied this from the readme. `git blame @baijum` :P

Comment (baijum) [reply manifests/devconsole/0.1.0/devconsole-operator.v0.1.0.clusterserviceversion.yaml]: @Avni-Sharma Good catch. @sbose78 Please update README also. Thanks!

Comment (sbose78) [reply manifests/devconsole/0.1.0/devconsole-operator.v0.1.0.clusterserviceversion.yaml]: Done!

**Comment by sbose78** [comment]
Thank you @alexeykazakov @baijum and @Avni-Sharma !

**Pull request merged**